### PR TITLE
WIP: Moving static to instance logging.

### PR DESCRIPTION
**Pull request by Hugos-MSFT**

### CODE REVIEW

File: src/Arriba/Arriba/Diagnostics/Tracing/ArribaLogs.cs [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        {
+            Console.WriteLine();
+        }
+
+        public void WriteLine(string message, params object[] args)
+        {
+            Console.WriteLine(message, args);
+        }
+
+        public IDisposable EnableConsoleOutput()
```
Comment (mattklup) [inline src/Arriba/Arriba/Diagnostics/Tracing/ArribaLogs.cs]: this seems to be not needed?

File: src/Arriba/Arriba/Diagnostics/Tracing/ArribaLogs.cs [hunk 2]
```diff
@@ -0,0 +1,26 @@
+using System;
+using System.Diagnostics;
+using System.Diagnostics.Tracing;
+
+namespace Arriba.Diagnostics.Tracing
+{
+    public class ArribaLogs
```
Comment (mattklup) [inline src/Arriba/Arriba/Diagnostics/Tracing/ArribaLogs.cs]: I would extract a small interface from this with the Write* methods and make the controllers and other stuff depend on that instead

File: src/Arriba/Tools/Arriba.TfsWorkItemCrawler/DefaultCrawler.cs [hunk 3]
```diff
@@ -32,14 +33,17 @@ public class DefaultCrawler
         private bool Rebuild { get; set; }
 
         private IEnumerable<string> ColumnNames { get; set; }
+        private ArribaLogging _log;
```
Comment (MS-Santi) [inline src/Arriba/Tools/Arriba.TfsWorkItemCrawler/DefaultCrawler.cs]: Where does this ArribaLogging type comes from?

File: src/Arriba/Arriba/Diagnostics/Tracing/ArribaLogs.cs (hunk 2 above)
Comment (MS-Santi) [inline src/Arriba/Arriba/Diagnostics/Tracing/ArribaLogs.cs]: Should probably be "ArribaLog" - singular

Comment (Hugos-MSFT) [reply src/Arriba/Arriba/Diagnostics/Tracing/ArribaLogs.cs]: Agree, I'll create a separate PR for this.

File: src/Arriba/Arriba/Diagnostics/Tracing/ArribaLogs.cs (hunk 1 above)
Comment (Hugos-MSFT) [reply src/Arriba/Arriba/Diagnostics/Tracing/ArribaLogs.cs]: Removed.

File: src/Arriba/Tools/Arriba.TfsWorkItemCrawler/DefaultCrawler.cs (hunk 3 above)
Comment (Hugos-MSFT) [reply src/Arriba/Tools/Arriba.TfsWorkItemCrawler/DefaultCrawler.cs]: Should be ArribaLog

File: src/Arriba/Arriba/Diagnostics/Tracing/ArribaLogs.cs (hunk 2 above)
Comment (Hugos-MSFT) [reply src/Arriba/Arriba/Diagnostics/Tracing/ArribaLogs.cs]: Done.

File: src/Arriba/Arriba.Server/Controllers/OAuthController.cs [hunk 4]
```diff
@@ -15,12 +16,13 @@ public class OAuthController : ControllerBase
         private readonly IOAuthConfig _config;
         private readonly HttpClient _http;
         private readonly IArribaServerConfiguration _serverConfig;
-
-        public OAuthController(IOAuthConfig config, IArribaServerConfiguration serverConfig)
+        private  ArribaLog _log;
```
Comment (ericmaino) [inline src/Arriba/Arriba.Server/Controllers/OAuthController.cs]: Make the field readonly

File: src/Arriba/Tools/Arriba.TfsWorkItemCrawler/DefaultCrawler.cs [hunk 5]
```diff
@@ -32,14 +33,17 @@ public class DefaultCrawler
         private bool Rebuild { get; set; }
 
         private IEnumerable<string> ColumnNames { get; set; }
+        private ArribaLog _log;
```
Comment (ericmaino) [inline src/Arriba/Tools/Arriba.TfsWorkItemCrawler/DefaultCrawler.cs]: readonly

File: src/Arriba/Tools/Arriba.WorkItemCrawler/DefaultCrawler.cs [hunk 6]
```diff
@@ -30,9 +31,10 @@ public class DefaultCrawler
         private bool Rebuild { get; set; }
 
         private IEnumerable<string> ColumnNames { get; set; }
-
-        public DefaultCrawler(CrawlerConfiguration config, IEnumerable<string> columnNames, string configurationName, bool rebuild)
+        private ArribaLog _log;
```
Comment (ericmaino) [inline src/Arriba/Tools/Arriba.WorkItemCrawler/DefaultCrawler.cs]: readonly

**Review by ericmaino — COMMENTED** [review]
Good progress. It would be great to update Trace.Write and Trace.WriteLine as well

File: src/Arriba/Arriba.Server/Controllers/OAuthController.cs (hunk 4 above)
Comment (Hugos-MSFT) [reply src/Arriba/Arriba.Server/Controllers/OAuthController.cs]: done.

File: src/Arriba/Tools/Arriba.TfsWorkItemCrawler/DefaultCrawler.cs (hunk 5 above)
Comment (Hugos-MSFT) [reply src/Arriba/Tools/Arriba.TfsWorkItemCrawler/DefaultCrawler.cs]: done.

**Pull request closed without merging**